### ISSUE TITLE
FastQ Screen: Plotting Issue 

### ISSUE THREAD

**Issue by skchronicles** [ewels/MultiQC#790]
**Description of bug:**
There seems to be an error when plotting the results of Fastq Screen. If `fastqscreen_simpleplot: false`, which it seems to be by default if the number of samples is low, the mapping of percent aligned data to genome type is incorrect. Here is an [example](https://imgur.com/a/Z6C2LBv) of what I am talking about:

**File that triggers the error:** [mQC.zip](https://github.com/ewels/MultiQC/files/2156978/mQC.zip)

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC:
    ```
    multiqc -f -c /path/to/multiqc_config.yaml --interactive -e cutadapt -d .
    ```
 - MultiQC Version: v1.4
 - Operating System: centOS7
 - Python Version: 2.7.14
 - Method of MultiQC installation: Unknown, installed on [Biowulf](https://hpc.nih.gov/apps/multiqc.html)

 **Additional context**
- Fixed by adding `fastqscreen_simpleplot: true` to `multiqc_config.yaml`

**Comment by ewels** [comment]
Thanks for the detailed report @skchronicles - this is of course a little concerning. I'll try to have a look into this later this week.

Phil

**Comment by skchronicles** [comment]
@ewels Just wondering if you have any updates

**Comment by skchronicles** [comment]
@ewels I found out what was causing the fastq_screen sample sorting problem. I noticed that when downloading multiqc from pypi via pip install, it is downloading an older version of multiqc that does not contain the  following change:
![image](https://user-images.githubusercontent.com/18038345/43158901-eb59bef2-8f4e-11e8-82fd-a2685c34d1d6.png)   
After adding the sorted() function to 'multiqc/modules/fastq_screen/fastq_screen.py', the problem was fixed.

**Comment by ewels** [comment]
Hi both,

Apologies, I was away on holiday. Ok yes, so basically it sounds like this bug has already been fixed (v1.4 is quite old now). Specifically, it looks like it was reported in #712 and fixed in #743

This fix is in the v1.6 development code, which will be released onto PyPI and conda very soon. You can install the development code in the short term if you need to (see instructions on the readme).

Thanks for reporting! Shout if you come across any other problems.

Phil
